youtube_dl/extractor/common.py

```diff
@@ -517,6 +517,13 @@ def _parse_json(self, json_string, video_id, transform_source=None, fatal=True):
             else:
                 self.report_warning(errmsg + str(ve))
 
+    def update_url_params(self, url, params):
+        parsed_url = compat_urlparse.urlparse(url)
+        qs = compat_urlparse.parse_qs(parsed_url.query)
```

Comment:
the code was modified from `CrunchyrollBaseIE._add_skip_wall` which use `compat_urlparse.parse_qs`.